packages/synapse-react-client/src/components/EntityFinder/tree/EntityTree.tsx

```diff
@@ -413,38 +413,22 @@ export function EntityTree(props: EntityTreeProps) {
       <div className="Header">
         <div className="Browse">Browse:</div>
         <div onClick={e => e.stopPropagation()}>
-          <Dropdown>
-            <Dropdown.Toggle variant="gray-primary-500" id="dropdown-basic">
-              {scope}
-            </Dropdown.Toggle>
-            <Dropdown.Menu role="menu">
-              {Object.values(FinderScope).map(scopeOption => {
-                if (
+          <DropdownSelect
+            variant={'outlined'}
+            options={Object.values(FinderScope).filter(
+              scopeOption =>
+                !(
                   scopeOption === FinderScope.CURRENT_PROJECT &&
                   projectId == null
-                ) {
-                  return <React.Fragment key={scopeOption} />
-                }
-                return (
-                  <Dropdown.Item
-                    role="menuitem"
-                    key={scopeOption}
-                    onClick={e => {
-                      e.stopPropagation()
-                      if (scope !== scopeOption) {
-                        setScope(scopeOption)
-                        setCurrentContainer(
-                          getScopeOptionDefaultContainer(scopeOption),
-                        )
-                      }
-                    }}
-                  >
-                    {scopeOption}
-                  </Dropdown.Item>
-                )
-              })}
-            </Dropdown.Menu>
-          </Dropdown>
+                ),
+            )}
+            selectedIndex={Object.values(FinderScope).indexOf(scope)}
```

Comment:
When `projectId` is undefined, `options` is a subset of `FinderValues`. However, `selectedIndex` is calculated using the full list of `FinderScope` values. Will the expected option be displayed? 

For example, when `initialScope` is `FinderScope.ALL_PROJECTS` and `projectId` is undefined, we would expect that 'All Projects' would be displayed.